website/client/src/components/auth/socialAuth/apple.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div
+    :class="{
+      'btn': !isHomePage,
+      'btn-secondary': !isHomePage,
+      'social-button': true,
+      'social-button-home': isHomePage,
+    }"
+    @click="socialAuth('apple')"
+  >
+    <div
+      class="svg-icon social-icon apple-icon"
+      v-html="icons.appleIcon"
+    ></div>
+    <span>{{ (registering || isHomePage || isGroupPlansPage)
+      ? $t('signUpWithSocial', {social: 'Apple'})
+      : $t('loginWithSocial', {social: 'Apple'}) }}</span>
```

Comment:
```suggestion
    <span>{{ $t(showSignUp ? 'signUpWithSocial' : 'loginWithSocial', {social: 'Apple'}) </span>
```

Reply:
@benblank  thx for this suggestion! indeed template should not contain much logic, so i add a computed property `showSignUp`.

Reply:
well..i delete this computed property... instead use a prop named `login`